Formula/htslib.rb

```diff
@@ -17,6 +17,11 @@ class Htslib < Formula
   uses_from_macos "curl"
   uses_from_macos "zlib"
 
+  patch do
```

Comment:
Can you add a comment to the formula file briefly explaining what this patch does and when it might be able to be removed (e.g. next release). This will allow future contributors to understand what this patch is at a quick glance.

Reply:
sure thing. this patch should be deleted on next upgrade of htslib and I made a comment to this effect

Reply:
not sure if this requires updating revision to this formula also

Reply:
Yes, the revision should be bumped.

Reply:
Added!